backend/src/main/java/com/map/configuration/DatabaseConnection.java

```diff
@@ -1,14 +1,16 @@
 package com.map.configuration;
 
-import org.apache.log4j.Logger;
+import com.map.repository.UserRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.context.annotation.Configuration;
 
 import java.sql.*;
 
 @Configuration
 public class DatabaseConnection {
 
-    private static final Logger log = Logger.getLogger(DatabaseConnection.class);
+    private static final Logger LOG = LoggerFactory.getLogger(UserRepository.class);
```

Comment:
in the log it will be `UserRepository.class` and should be the class where it was logged